controllers/preflightvalidation_controller_test.go

```diff
@@ -0,0 +1,315 @@
+package controllers
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/golang/mock/gomock"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	kmmv1beta1 "github.com/qbarrand/oot-operator/api/v1beta1"
+	"github.com/qbarrand/oot-operator/internal/client"
+	"github.com/qbarrand/oot-operator/internal/preflight"
+	"github.com/qbarrand/oot-operator/internal/statusupdater"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+const (
+	preflightName = "test-preflight"
+)
+
+var _ = Describe("PreflightValidationReconciler", func() {
```

Comment:
```suggestion
var _ = Describe("Reconcile", func() {
```

I suggest we keep the `Describe` name identical with the function/method name it is testing.

Reply:
fixed